[2022-12-04T16:14:38.689+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T11:00:00+00:00 [queued]>
[2022-12-04T16:14:38.697+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-04T11:00:00+00:00 [queued]>
[2022-12-04T16:14:38.699+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:38.700+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:38.701+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:38.711+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-04 11:00:00+00:00
[2022-12-04T16:14:38.719+0000] {standard_task_runner.py:55} INFO - Started process 590 to run task
[2022-12-04T16:14:38.724+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-04T11:00:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpguky8ru1']
[2022-12-04T16:14:38.727+0000] {standard_task_runner.py:83} INFO - Job 74: Subtask notification
[2022-12-04T16:14:38.803+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-04T11:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:38.865+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T11:00:00+00:00
[2022-12-04T16:14:38.868+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:14:38.869+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-04']
[2022-12-04T16:14:38.884+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:14:38.887+0000] {subprocess.py:93} INFO - received data at 2022-12-04
[2022-12-04T16:14:38.888+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:14:38.915+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221204T110000, start_date=20221204T161438, end_date=20221204T161438
[2022-12-04T16:14:38.976+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:38.997+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
